lnrpc/rpc.proto

```diff
@@ -647,6 +657,24 @@ service Lightning {
     };
 }
 
+message Utxo {
+
+	/// The type of address
+	string address_type = 1 [json_name = "address_type"];
+
+	/// The address
+	string address = 2 [json_name = "address"];
+
+	/// the value of the unspent coin in satoshis
+	int64 amount = 3 [json_name = "amount"];
```

Comment:
should be named `amount_sat`

Reply:
Done in 60ae2afc236c6eb3d1095cca9dfed5d9760a23b4